FirebaseRemoteConfig/Sources/RCNFetch.m

```diff
@@ -111,6 +109,9 @@ - (instancetype)initWithContent:(RCNConfigContent *)content
 
 /// Force a new NSURLSession creation for updated config.
 - (void)recreateNetworkSession {
+  if (self.testWithoutNetwork) {
```

Comment:
What if we inject implementation of `newFetchSession` method (e.g. with a block) as instead? Or we can use OCMock to stub `+ [NSURLSession sessionWithConfiguration:]` method to avoid the class changes altogether.

Reply:
This one is also tricky, since we still need the first `newFetchSession` to happen before any mocking. Thus, I went with the more expedient mock of `recreateNetworkSession` until we do a bigger refactor.